server/pom.xml

```diff
@@ -197,6 +197,18 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.candlepin</groupId>
+      <artifactId>api</artifactId>
+      <version>unspecified</version>
```

Comment:
This is more relevant to the spec-first branch/feature than to this, but shouldn't this have a version here? Do/Should we declare the api as a maven module?